src/main/java/racingcar/controller/WebController.java

```diff
@@ -1,32 +1,39 @@
 package racingcar.controller;
 
-import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
+import racingcar.dto.*;
+import racingcar.services.RacingGameService;
+import racingcar.util.ValueEditor;
 
-import racingcar.dto.GameInfoDto;
-import racingcar.dto.ResultDto;
-import racingcar.services.GameService;
+import java.util.List;
 
 @RestController
 public class WebController {
 
-    @Autowired
-    private final GameService gameService;
+    private final RacingGameService racingGameService;
 
-    public WebController(GameService gameService) {
-        this.gameService = gameService;
+    public WebController(RacingGameService racingGameService) {
+        this.racingGameService = racingGameService;
     }
 
     @PostMapping("/plays")
-    public ResultDto play(@RequestBody GameInfoDto gameInfoDto) {
-        gameService.initialize();
-        gameService.createCars(gameInfoDto.getNames());
-        gameService.moveCars(gameInfoDto.getCount());
-        ResultDto resultDto = new ResultDto(gameService.getWinner(), gameService.getResult());
-        gameService.saveResult(gameInfoDto.getCount(), resultDto);
-        return resultDto;
+    public ResponseEntity<ResponseDto> play(@RequestBody RequestDto requestDto) {
+        List<String> names = ValueEditor.splitByComma(requestDto.getNames());
+        String count = requestDto.getCount();
+        RacingGameDto racingGameDto = racingGameService.play(new StartInformationDto(names, count));
```

Comment:
```suggestion
        RacingGameDto racingGameDto = racingGameService.play(new StartInformationDto(names, requestDto.getCount()));
```
따로 지역변수로 뺄 필요가 있을까요?

Reply:
따로 뻴 필요는 없는 것 같아요!
그렇지만 파라미터를 names, count 이렇게 넘기는게 한눈에 보기 좋아서 한번 빼봤습니다